src/main/kotlin/com/exactpro/th2/processor/core/HandleMessageException.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022-2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.processor.core
+
+import com.exactpro.th2.common.grpc.MessageID
+
+@Suppress("unused")
+open class HandleMessageException : ProcessorException {
+    val messageIds: List<MessageID>
+
+    constructor(messageIds: List<MessageID>) : super() { this.messageIds = messageIds }
+    constructor(message: String?, messageIds: List<MessageID>) : super(message) { this.messageIds = messageIds }
+    constructor(message: String?, messageIds: List<MessageID>, cause: Throwable?) : super(message, cause) { this.messageIds = messageIds }
+    constructor(messageIds: List<MessageID>, cause: Throwable?) : super(cause) { this.messageIds = messageIds }
+    constructor(message: String?, messageIds: List<MessageID>, cause: Throwable?, enableSuppression: Boolean, writableStackTrace: Boolean) : super(
+        message,
+        cause,
+        enableSuppression,
+        writableStackTrace
+    ) { this.messageIds = messageIds }
+}
```

Comment:
This way it looks better, I think
```suggestion
open class HandleMessageException(
  val messageIds: List<MessageID>,
  message: String? = null,
  cause: Throwable? = null,
  enableSuppression: Boolean = true,
  writableStackTrace: Boolean = true,
) : ProcessorException(
        message,
        cause,
        enableSuppression,
        writableStackTrace,
)
```